test/quantization/core/test_quantized_module.py

```diff
@@ -72,22 +73,28 @@ def test_linear_api(self):
             [16, 32],
             [4, 8],
             [True, False],
-            [True, False],
+            ['relu', 'none'],
             [True, False])
         for (batch_size, in_features, out_features, use_bias,
-             use_fused, per_channel) in options:
+             post_op, per_channel) in options:
             self._test_linear_api_impl(
-                batch_size, in_features, out_features, use_bias, use_fused,
+                batch_size, in_features, out_features, use_bias, post_op,
                 per_channel)
 
-    def _test_linear_api_impl(self, batch_size, in_features, out_features, use_bias, use_fused, per_channel):
+    def _test_linear_api_impl(self, batch_size, in_features, out_features, use_bias, post_op, per_channel, **post_ops_kwargs):
         if torch.backends.quantized.engine == 'qnnpack':
             per_channel = False
 
         # use_fused -> quantized class
         class_map = {
-            True: nniq.LinearReLU,
-            False: nnq.Linear,
+            'none': nnq.Linear,
+            'relu': nniq.LinearReLU,
+            'leaky_relu': nniq.LinearLeakyReLU,
+        }
+        q_module_name_map = {
+            'none': 'QuantizedLinear',
+            'relu': 'QuantizedLinearReLU',
+            'leaky_relu': 'QuantizedLinearLeakyReLU',
         }
```

Comment:
I think we should probably pass these from argument, it looks a bit weird that we hardcode these in this helper function

Reply:
also for naming, I think we can rename `test_linear_api` to `test_linear` as well, to make it clearer

Reply:
and we can split `test_linear` to `test_linear` and `test_linear_relu` to make sure each test only tests one thing

Reply:
OK I will do that.

Reply:
I have made these as arguments of the helper function and split `test_qlinear` to two tests for `qlinear` and `qlinear_relu` respectively.